ui/package.json

```diff
@@ -21,10 +21,10 @@
         "watch:server": "webpack --config ./__config__/webpack.config.dev.js --env.server --watch"
     },
     "dependencies": {
+        "emotion": "^7.3.2",
```

Comment:
This is now a duplicated entry ... https://github.com/guardian/frontend/blob/f1dfc10210f9f2f9360a162ccc322435d5e68169/ui/package.json#L50

Reply:
removed from devdeps but not showing